sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosBulkAsyncTest.java

```diff
@@ -127,6 +154,10 @@ public void createItemMultipleTimesWithOperationOnFly_withBulk() {
                 processedDoc.incrementAndGet();
 
                 CosmosBulkItemResponse cosmosBulkItemResponse = cosmosBulkOperationResponse.getResponse();
+                if (cosmosBulkOperationResponse.getException() != null) {
```

Comment:
nit: since we are in tests, so maybe using assert?

Reply:
No - I really want the exception details to be in the log.